backend/functions/lib/controllers/orders/orders.controller.test.ts

```diff
@@ -81,18 +84,39 @@ describe('orders controller tests', () => {
     expect(syncOrdersSpy).toHaveBeenCalledWith(applicationContext, undefined);
   });
 
+  test('should get suggested cases', async () => {
+    const suggestedCases = [CASE_SUMMARIES[0]];
+    const suggestedCasesResponse: CamsResponse<CaseDetailInterface[]> = {
+      body: suggestedCases,
+      success: true,
+    };
+
+    const getSuggestedCasesSpy = jest
+      .spyOn(OrdersUseCase.prototype, 'getSuggestedCases')
+      .mockResolvedValue(suggestedCases);
+
+    const controller = new OrdersController(applicationContext);
+    const response = await controller.getSuggestedCases(applicationContext, 'mockId');
+    expect(getSuggestedCasesSpy).toHaveBeenCalledWith(applicationContext, 'mockId');
+    expect(response).toEqual(suggestedCasesResponse);
+  });
+
   test('should rethrow CamsError if CamsError is ecountered', async () => {
     const camsError = new CamsError('TEST');
     jest.spyOn(OrdersUseCase.prototype, 'getOrders').mockRejectedValue(camsError);
     jest.spyOn(OrdersUseCase.prototype, 'updateOrder').mockRejectedValue(camsError);
     jest.spyOn(OrdersUseCase.prototype, 'syncOrders').mockRejectedValue(camsError);
+    jest.spyOn(OrdersUseCase.prototype, 'getSuggestedCases').mockRejectedValue(camsError);
 
     const controller = new OrdersController(applicationContext);
     await expect(controller.getOrders(applicationContext)).rejects.toThrow(camsError);
     await expect(controller.updateOrder(applicationContext, id, orderTransfer)).rejects.toThrow(
       camsError,
     );
     await expect(controller.syncOrders(applicationContext)).rejects.toThrow(camsError);
+    await expect(controller.getSuggestedCases(applicationContext, 'mockId')).rejects.toThrow(
```

Comment:
I've noticed we use error types such as CamsError in expects for error thrown but plain text strings in others. Shouldn't we be consistent in this? 

For example we are throwing a NotFoundError above in case-history.controller.test.ts but running the expect to match the returned message string,  and here we are testing against an instance of CamsError

Reply:
Agreed.  I didn't realize we could just pass in a class before.  I think I like that better.

Reply:
In this case, we're passing in an instance of a camsError.  But I noticed earlier today we used toThrow() and we passed it a class Type.  i.e. we passed in toThrow(CamsError) with a capital C.  

Is creating an instance unneccessary?

Reply:
Punting for now.